ui/src/dataLoaders/components/configureStep/ScraperTarget.tsx

```diff
@@ -28,11 +30,21 @@ export class ScraperTarget extends PureComponent<Props> {
   }
 
   public render() {
-    const {onSelectBucket, url, bucket, buckets} = this.props
+    const {onSelectBucket, url, name, bucket, buckets} = this.props
     return (
       <Grid>
         <Grid.Row>
           <Grid.Column widthXS={Columns.Eight} offsetXS={Columns.Two}>
+            <Form.Element label="Name">
+              <Input
```

Comment:
is a name allowed to be empty?

Reply:
the api does not complain if name is empty.  i could add a check for it and give it a default name (Name this Scraper target) like we do with telegraf configs.

Reply:
a default or as-is seems good to me

Reply:
i have added a default. it works the same as the scraper url now.